Dockerfile

```diff
@@ -1,21 +1,14 @@
-FROM quay.io/pypa/manylinux1_x86_64 as wheel_builder
+FROM python:3.7.1-stretch
 
-COPY ./pairing /usr/src/pairing
-#RUN git clone -b docker-build https://github.com/sbellem/pairing.git /usr/src/pairing
-WORKDIR /usr/src/pairing
-
-RUN sh scripts/build-wheel.sh
+ENV PYTHONUNBUFFERED=1
 
+RUN apt-get update && apt-get install -y vim tmux
```

Comment:
I know that the tests are passing but just curious - are `make` and `gcc` available by default in this image?

Reply:
yes